index.html

```diff
@@ -21,20 +21,20 @@ <h1 class="headin">I’m Brenda.Glad to see you!</h1>
         website Look through some of my work and experience! If you like what
```

Comment:
- [ OPTIONAL]i Can see that your Design didn't reflect the Figma design , I just noticed that your background-image is not well fixed , kindly fix it for your design to match the Figma design  . check on this image below the difference :point_down: , your logo, your fonts, colors are not really the same 
Your Design 
![Screenshot from 2023-03-31 22-16-18](https://user-images.githubusercontent.com/88894525/229233408-c5886870-398a-4b24-81cb-f03b65595a48.png)

Figma 
![Screenshot from 2023-03-31 23-18-43](https://user-images.githubusercontent.com/88894525/229233290-68238fea-2d70-4f86-87b9-879fe42eabf9.png)